.github/workflows/nightly.yml

```diff
@@ -0,0 +1,55 @@
+name: Nightly
+
+on:
+  workflow_dispatch:
+  schedule:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    # 12:15 UTC, every day
+    - cron: "15 12 * * *"
```

Comment:
Ok, so "nightly" means… slightly after noon 😆  I guess it's always night somewhere

Reply:
I was thinking this way we get to react to any build failures